src/style.css

```
@@ -0,0 +1,31 @@
+body {
+  background-color: black;
+}
+
+.txt {
+  background-color: #ddd;
+  border: 1px solid #e6635f;
+  width: 284px;
+  margin: 0;
+  padding: 10px;
+}
+
+.top,
+#qa-block {
```

```suggestion
.top {
```
they links to the same element